src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/contactperson/ContactPersonDpsApiMockServer.kt

```diff
@@ -28,7 +29,10 @@ class ContactPersonDpsApiExtension :
     @JvmField
     val dpsContactPersonServer = ContactPersonDpsApiMockServer()
     lateinit var objectMapper: ObjectMapper
+
+    @Suppress("unused")
     inline fun <reified T> getRequestBody(pattern: RequestPatternBuilder): T = dpsContactPersonServer.getRequestBody(pattern, objectMapper)
```

Comment:
I'll keep this since it asserts that is at least one request find - so is of use as a convenience when you know there should be just one request made to a stub